cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/TestNativeImageCommandTest.java

```diff
@@ -66,20 +71,30 @@ public void testNativeImageTestsWithAdditionalArgs() throws IOException {
         Path projectPath = this.testResources.resolve("validProjectWithTests");
         System.setProperty(ProjectConstants.USER_DIR, projectPath.toString());
         TestCommand testCommand = new TestCommand(projectPath, printStream, printStream, false, true, "-H:Name=foo");
-        new CommandLine(testCommand).parse();
-        testCommand.execute();
+        new CommandLine(testCommand).parseArgs();
+        try {
+            testCommand.execute();
+        } catch (BLauncherException e) {
+            readOutput(false);
+            Assert.fail(e.getDetailedMessages().get(0));
+        }
         String buildLog = readOutput(true);
-        Assert.assertTrue(buildLog.contains("Generating 'foo' (executable)"));
+        Assert.assertTrue(buildLog.contains("Generating 'foo' (executable)"), buildLog);
```

Comment:
@Shadow-Devil Sorry for the delay. I couldn't go through this PR due to some other urgent work. Today I checked the reason for the failure. We need to change this assertion as,
```ballerina
Assert.assertTrue(buildLog.contains("Generating 'winery' (executable)"), buildLog);
```

This is a failing test case that needs to be fixed. For some reason, we couldn't detect this test failure earlier.

Reply:
No worries, I know that this is only low priority and you have other tasks to do. Thank you for taking the time to check this 😄